docs/user/04-metrics.md

```diff
@@ -491,15 +491,16 @@ To activate the constructed MetricPipeline, follow these steps:
 
 ### Result
 
-You activated a MetricPipeline and metrics start streaming to your backend. To verify that the pipeline is running, verify that the status of the LogPipeline in your cluster is `Ready`:
+You activated a MetricPipeline and metrics start streaming to your backend. To verify that the pipeline is running, verify that the status of the MetricPipeline in your cluster is `Ready`:
     ```bash
     kubectl get metricpipeline
     NAME              STATUS    AGE
     backend           Ready     44s
+    ```
 
 ## Operations
 
-A MetricPipeline creates a Deployment running OTel Collector instances in your cluster. That instances will serve OTLP endpoints and ship received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop logs because the backend is either not reachable for some duration, or cannot handle the log load and is causing back pressure.
+A MetricPipeline creates a Deployment running OTel Collector instances in your cluster. That instance serves OTLP endpoints and ships received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop metrics because the backend is either not reachable for some duration, or cannot handle the metrics load and is causing back pressure.
```

Comment:
```suggestion
A MetricPipeline creates a Deployment running OTel Collector instances in your cluster. That Deployment serves OTLP endpoints and ships received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop metrics because the backend is either not reachable for some duration, or cannot handle the metrics load and is causing back pressure.
```